test/unit/Assessment/startAssessment.js

```diff
@@ -125,4 +125,28 @@ describe('startAssessment', function () {
       expect(poll.denied).to.be.equal(0);
     }
   });
+
+  it('reverts if caller is not an internal contract', async function () {
+    const { assessment } = this.contracts;
+    const [user] = this.accounts.members;
+    const [AB] = this.accounts.advisoryBoardMembers;
+    const [governance] = this.accounts.governanceContracts;
+    const admin = this.accounts.emergencyAdmin;
+
+    await expect(assessment.connect(admin).startAssessment(parseEther('100'), parseEther('10'))).to.be.revertedWith(
```

Comment:
the property being verified here is that it reverts if a non-internal contract call reverts

We don't need to add expect for multiple callers, only something that is nonInternal

1 expect suffices